mypy/messages.py

```diff
@@ -872,6 +872,9 @@ def unimported_type_becomes_any(self, prefix: str, typ: Type, ctx: Context) -> N
         self.fail("{} becomes {} due to an unfollowed import".format(prefix, self.format(typ)),
                   ctx)
 
+    def explicit_any(self, location: str, ctx: Context) -> None:
+        self.fail('Explicit "Any" is not allowed in a {}'.format(location), ctx)
```

Comment:
Yeah, actually, I agree. I think I was trying to be too fancy when it wasn't necessary.